ArgusCore/src/main/java/com/salesforce/dva/argus/entity/SuspensionLevel.java

```diff
@@ -0,0 +1,223 @@
+package com.salesforce.dva.argus.entity;
+
+import static com.salesforce.dva.argus.system.SystemAssert.requireArgument;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityManager;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.NoResultException;
+import javax.persistence.Table;
+import javax.persistence.TypedQuery;
+import javax.persistence.UniqueConstraint;
+
+/**
+ * The entity encapsulates information about the suspension levels for a given policy.
+ *
+ * <p>Fields that determine uniqueness are:</p>
+ *
+ * <ul>
+ *   <li>ID</li>
```

Comment:
technically, it should be policy_id and level_number.
level number should also be greater than 0 (enforce this on setLevel(...))